averbis/core/_rest_client.py

```diff
@@ -248,6 +318,39 @@ def get_info(self) -> dict:
         # noinspection PyProtectedMember
         return self.project.client._get_pipeline_info(self.project.name, self.name)
 
+    @experimental_api
+    def list_resource_containers(self) -> List[ResourceContainer]:
+        """
+        HIGHLY EXPERIMENTAL API - may soon change or disappear.
+
+        List the resource containers for the current pipeline.
+        """
+        pipeline_url = f"/experimental/textanalysis/projects/{self.project.name}/pipelines/{self.name}"
+        # noinspection PyProtectedMember
+        return self.project.client._list_resource_containers(pipeline_url)
+
+    @experimental_api
+    def create_resource_container(self, name: str, resources_zip_path: Optional[Union[Path, str]] = None) -> ResourceContainer:
+        """
+        HIGHLY EXPERIMENTAL API - may soon change or disappear.
+
+        Create empty resource container with given name for this pipeline or additionally upload zipped resources.
+        """
+        pipeline_url = f"/experimental/textanalysis/projects/{self.project.name}/pipelines/{self.name}"
+        # noinspection PyProtectedMember
+        return self.project.client._create_resource_container(name, pipeline_url, resources_zip_path)
+
+    @experimental_api
+    def delete_resource_container(self, name: str) -> None:
```

Comment:
We normally have the `delete` endpoints on the object itself, e.g. `pipeline.delete()` or `project.delete()`. In that sense, we should have a `ResourceContainer.delete()` method.